packages/theme/src/popover/responsive.less

```
@@ -0,0 +1,24 @@
+@import '../custom.less';
+@import './vars.less';
+
+@popover-prefix-cls: ~'@{css-prefix}popover';
+@popper-prefix-cls: ~'@{css-prefix}popper';
+
+@media screen and (max-width: 1280px) {
+  .@{popover-prefix-cls}.@{popper-prefix-cls} {
```

Stylelint (17.4.0)

[error] Unknown word popover-prefix-cls (CssSyntaxError)

(CssSyntaxError)